packages/manager/modules/support/src/models/ticket/message/message.class.js

```diff
@@ -0,0 +1,11 @@
+import { FROM } from './message.constants';
+
+export default class {
```

Comment:
👍

Reply:
Noticed that sometimes this file is named as `TitleCase` (eg `packages/manager/modules/pci/src/projects/Project.class.js`).

Should we have to rename it as `Message.class.js` instead? (in order to Harmonize things)

Reply:
Could be addressed later since this pull request receive the "quality check ok" label.